src/functions/getConfiguration/domain/getGAId.ts

```diff
@@ -0,0 +1 @@
+export const getGAId = (): string => process.env.GOOGLE_ANALYTICS_ID || 'UA-129489007-3';
```

Comment:
Can the Id after the Or be assigned as a const? Could be considered a magic string?

Reply:
Think all environment variables are magic strings by nature! This just provides a fallback to the dev ID

Reply:
No problem. Thank you for explaining :)